infrastructure/stacks/capacity_management/lambda.tf

```diff
@@ -0,0 +1,319 @@
+#    }
+
+/**************************
+//
+//LOCATIONS DATA MANAGER LAMBDA
+//
+**************************/
+module "locations-data-manager-lambda" {
+  source = "../../modules/lambda"
+
+  function_name = "locations-data-manager"
+  description   = "Microservice for interacting with locations dynamodb table"
+
+  policy_jsons = [
+    <<-EOT
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "DynamodbTable",
+                    "Effect": "Allow",
+                    "Action": [
+                        "dynamodb:PutItem",
+                        "dynamodb:DeleteItem",
+                        "dynamodb:GetItem",
+                        "dynamodb:Scan",
+                        "dynamodb:Query",
+                        "dynamodb:UpdateItem"
+                    ],
+                    "Resource": [
+                        "${module.dynamodb_locations_table.dynamodb_table_arn}"
+                    ]
+                }
+            ]
+        }
+        EOT
+  ]
+}
+
+#    allowed_triggers = {
+#        AllowExecutionFromAPIGateway = {
+#        service    = "apigateway"
+#        source_arn = "${aws_api_gateway_rest_api.DoS_REST.execution_arn}/*/*"
+#        }
+#    }
+#    }
+
+#    module "live-alias-locations-data-manager" {
+#    source = "terraform-aws-modules/lambda/aws//modules/alias"
+
+#    name          = "live-service"
+#    function_name = module.locations-data-manager-lambda.lambda_function_name
+#    refresh_alias = false
+#    }
+
+/**************************
+//
+//HEALTHCARE SERVICES DATA MANAGER LAMBDA
+//
+**************************/
+module "healthcare-services-data-manager-lambda" {
+  source = "../../modules/lambda"
+
+  function_name = "healthcare-services-data-manager"
+  description   = "Microservice for interacting with healthcare services dynamodb table"
+
+  policy_jsons = [
+    <<-EOT
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "DynamodbTable",
+                    "Effect": "Allow",
+                    "Action": [
+                        "dynamodb:PutItem",
+                        "dynamodb:DeleteItem",
+                        "dynamodb:GetItem",
+                        "dynamodb:Scan",
+                        "dynamodb:Query",
+                        "dynamodb:UpdateItem"
+                    ],
+                    "Resource": [
+                        "${module.dynamodb_healthcare_services_table.dynamodb_table_arn}"
+                    ]
+                }
+            ]
+        }
+        EOT
+  ]
+}
+
+#    allowed_triggers = {
+#        AllowExecutionFromAPIGateway = {
+#        service    = "apigateway"
+#        source_arn = "${aws_api_gateway_rest_api.DoS_REST.execution_arn}/*/*"
+#        }
+#    }
+#    }
+
+#    module "live-alias-healthcare-services-data-manager" {
+#    source = "terraform-aws-modules/lambda/aws//modules/alias"
```

Comment:
See similar comments?

Reply:
Needed as part of the work coming up in the next sprint ..same as above